runtime/doc/map.txt

```diff
@@ -1431,6 +1431,43 @@ Possible values are (second column is the short name used in listing):
     -addr=other		  ?	other kind of range
 
 
+Incremental preview ~
+                                                  *:command-preview*
+It's possible to allow commands to be incrementally previewed when
+'inccommand' is enabled by using the -preview command attribute. Possible use
```

Comment:
Do you mean if `inccommand` is set to `split`? I don't think `inccommand` is a boolean option, it seems that the only possible values are `nosplit` (default) and `split`.

Not even sure why it's not a boolean called `incsplit` tbh.

Reply:
`'inccommand'` can also be set to an empty string.

Reply:
This applies to both `inccommand=nosplit` and `inccommand=split`

Reply:
That's even worse. Impressive.

Reply:
I see. The docs for `inccommand` confused me.